modules/client/src/addressList/interfaces.ts

```diff
@@ -103,10 +103,10 @@ export type SubgraphAddressListProposalListItem = SubgraphProposalBase;
 export type SubgraphAddressListProposal = SubgraphProposalBase & {
   createdAt: string;
   actions: SubgraphAction[];
-  supportRequiredPct: string;
-  participationRequired: string;
+  totalSupportThresholdPct: string;
+  relativeSupportThresholdPct: string;
   voters: SubgraphAddressListVoterListItem[];
-  votingPower: string;
+  census: string;
 };
```

Comment:
These are Subgraph's types, right? (they are going to change soon)

Reply:
yes, when they change i'll change it again